lib/rules/jsdoc.js

```diff
@@ -336,11 +381,11 @@
  *
  * Reports invalid types for bunch of tags
  *
- * In `strictNativeCase` mode ensures that case of natives is the same as in
- * this list: `boolean`, `number`, `string`, `Object`, `Array`, `Date`, `RegExp`.
+ * In `strictNativeCase` mode ensures that case of natives is the same as in this
+ * list: `boolean`, `number`, `string`, `Object`, `Array`, `Date`, `RegExp`.
  *
- * In `capitalizedNativeCase` mode ensures that first letter in all native types and
- * primitives is uppercased except the case with `function` in google closure
+ * In `capitalizedNativeCase` mode ensures that first letter in all native types
```

Comment:
`In capitalizedNativeCase mode` -> `The capitalizedNativeCase mode`

`that first letter ` -> `that the first letter `